proto/osmosis/epochs/genesis.proto

```diff
@@ -27,7 +27,6 @@ message EpochInfo {
     (gogoproto.moretags) = "yaml:\"current_epoch_start_time\""
   ];
   bool epoch_counting_started = 6;
-  bool current_epoch_ended = 7;
```

Comment:
Can we keep this field, just to not break the proto file, and rename the variable to `deprecated_current_epoch_ended`

Reply:
Later we should figure out how to improve that, but I think we can punt on that state migration logic.

Reply:
Ah wait, theres a thing in protobuf for this. We can delete this field, but we need to add a line for `reserved 7`. https://developers.google.com/protocol-buffers/docs/proto3#reserved